securebanking-openbanking-uk-rs-simulator-server/src/main/java/com/forgerock/securebanking/openbanking/uk/rs/configuration/RsApplicationConfiguration.java

```diff
@@ -73,7 +77,12 @@ public Jackson2ObjectMapperBuilderCustomizer objectMapperBuilderCustomizer() {
             jacksonObjectMapperBuilder.featuresToDisable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
             jacksonObjectMapperBuilder.featuresToEnable(MapperFeature.USE_BASE_TYPE_AS_DEFAULT_IMPL);
             jacksonObjectMapperBuilder.modules(new JodaModule());
-            jacksonObjectMapperBuilder.dateFormat(new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZZ"));
+            jacksonObjectMapperBuilder.featuresToEnable(DeserializationFeature.USE_BIG_DECIMAL_FOR_FLOATS);
+            jacksonObjectMapperBuilder.serializerByType(BigDecimal.class, new CustomObjectMapper.BigDecimalSerializer(BigDecimal.class));
```

Comment:
Is this configuring the jackson settings for the rs-simulator-server as a whole?

If so, are the BigDecimalSerializer/Deserializer behaving appropriately for all BigDecimals that we serialize? I'm not aware where BigDecimals are currently being used in the Open Banking Spec but I know for things like FX rates, banks tend to quote > 2dp

Reply:
I removed the BigDecimalSerializer/Deserializer from the jacksonObjectMapperBuilder and moved the serialization/deserialization to my customObjetcMapper.